src/utils/source_ip_check.ts

```diff
@@ -162,5 +163,12 @@ export function clientIPAndCidrTuple(
   clientIp: ClientIp,
   userAttributes: IAzureUserAttributes
 ): ITuple2<ClientIp, ReadonlySet<string>> {
-  return Tuple2(clientIp, userAttributes.service.authorizedCIDRs);
+  return Tuple2(
+    clientIp,
+    toAuthorizedCIDRs(
+      Array.from(userAttributes.service.authorizedCIDRs).map(c =>
+        c.indexOf("/") !== -1 ? c : `${c}/32`
+      )
```

Comment:
We should be more explicit about what's going on. We can either add a line of comment or write something like
```ts
const withDefaultSubnet = ip => ip.indexOf("/") !== -1 ? ip : `${ip}/32

[...]

Array.from(userAttributes.service.authorizedCIDRs).map(withDefaultSubnet)
```

Reply:
fixed in 7211251